### PR TITLE
Remove copy/paste errors that leaked `Podman` into man pages.

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: docs/buildah-bud.md [hunk 1]
```diff
@@ -316,12 +316,14 @@ Add an image *label* (e.g. label=*value*) to the image metadata. Can be used mul
 Users can set a special LABEL **io.containers.capabilities=CAP1,CAP2,CAP3** in
 a Containerfile that specified the list of Linux capabilities required for the
 container to run properly. This label specified in a container image tells
-Podman to run the container with just these capabilities. Podman launches the
-container with just the specified capabilities, as long as this list of
-capabilities is a subset of the default list.
+container engines, like Podman, to run the container with just these
+capabilities. Podman launches the container with just the specified
```
Comment (TomSweeneyRedHat) [inline docs/buildah-bud.md:320]: Should Podman be changed in the second line to:
```suggestion
capabilities. The container engine launches the container with just the specified
```

**Comment by TomSweeneyRedHat** [comment]
One question, otherwise LGTM

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Review by ashley-cui — COMMENTED** [review]
LGTM

**Pull request merged**